xdp-dump/test-xdpdump.sh

```diff
@@ -62,6 +64,10 @@ Options:
 END
 	  )
 
+    if grep -q "HAVE_LIBBPF_PERF_BUFFER__CONSUME:=n" "$CONFIG_MK"; then
```

Comment:
I would like to make the tests runnable from the installed RPM. In which case config.mk won't be available. So can we come up with a better way to detect this?

Some ideas:
- grep the --help output
- have the Makefile write a test config that we can install along with the test scripts

Reply:
Didn't see any resolution of this?

Reply:
I missed this comment, let me change this, and ping you!